test/confs/program_output.yaml

```diff
@@ -41,4 +41,4 @@ stdout_output:
 
 program_output:
   enabled: true
-  program: cat > /tmp/falco_outputs/program_output.txt
+  program: cat >> /tmp/falco_outputs/program_output.txt
```

Comment:
This command keeps a process open, that's what you want?

Reply:
It's not really important, Falco will close the program :point_down: 

https://github.com/falcosecurity/falco/blob/8b56360f8ca2f088c27ac124b01dccdce8ec5bc6/userspace/falco/lua/output.lua#L136